pinot-spi/src/main/java/org/apache/pinot/spi/env/CommonsConfigurationUtils.java

```diff
@@ -84,9 +90,10 @@ public static PropertiesConfiguration fromPath(String path)
    * @param setDefaultDelimiter representing to set the default list delimiter.
    * @return a {@link PropertiesConfiguration} instance.
    */
-  public static PropertiesConfiguration fromPath(String path, boolean setIOFactory, boolean setDefaultDelimiter)
+  public static PropertiesConfiguration fromPath(String path, boolean setIOFactory, boolean setDefaultDelimiter,
```

Comment:
looks like those fromXXX(, ... PropertyIOFactoryKind ioFactoryKind) can be `private` methods for now?

Reply:
Well this is the util class, I think having this as public make sense.